cmds/signin/signin.go

```diff
@@ -111,17 +109,23 @@ func cmdSignin(cmd *cobra.Command, _ []string) error {
 	}
 
 	// Construct URL for login service and open it
-	target := "http://" + strings.Replace(listener.Addr().String(), "127.0.0.1", "localhost", 1)
-	description := url.QueryEscape(
-		`Login and click the "Grant Access" button to transfer
-		temporary credentials to the taskcluster CLI client`,
-	)
-	loginURL := config.Configuration["signin"]["loginUrl"].(string)
-	loginURL += "/?target=" + url.QueryEscape(target) + "&description=" + description
+	callbackURL := "http://" + strings.Replace(listener.Addr().String(), "127.0.0.1", "localhost", 1)
+	description := url.QueryEscape("Temporary client for use on the command line")
+	loginURL := config.Configuration["signin"]["toolsUrl"].(string) + "/auth/clients/new";
+	name, _ := cmd.Flags().GetString("name")
+	loginURL += "?name=" + url.QueryEscape(name)
+	loginURL += "&description=" + description
+	scopes, _ := cmd.Flags().GetStringArray("scope")
+	for i := range scopes {
+		loginURL += "&scope=" + scopes[i]
+	}
+	expires, _ := cmd.Flags().GetString("expires")
+	loginURL += "&expires=" + url.QueryEscape(expires)
+	loginURL += "&callback_url=" + url.QueryEscape(callbackURL)
 
 	// Open browser
-	fmt.Fprintln(cmd.OutOrStdout(), "Listening for a callback on: "+target)
-	fmt.Fprintln(cmd.OutOrStdout(), "Opening URL: "+loginURL)
+	fmt.Fprintln(cmd.OutOrStderr(), "Listening for a callback on: "+callbackURL)
```

Comment:
So one would write: 
```bash
alias taskcluster-login="eval `taskcluster login`"
```

even better:
```bash
taskcluster() {
    if [[ $@ == "-la" ]]; then
        eval `taskcluster login`
    else
        taskcluster "$@"
    fi
}
```

Reply:
Could we make an argument like `taskcluster login --setup` that will install this in `/etc/bash_completion.d/` or maybe we can integrate it with the bash completion we have.. (not sure if this abuse of bash_completion).

Or at-least have a section in `taskcluster help login` that explains how to create these and add them to `.bashrc`

Reply:
I don't want to mess with shell completion.  And that function needs some work to properly find the path to taskcluster and not invoke itself recursively (and also to use `taskcluster signin` not `taskcluster login`).  I think an alias is simpler.